### DISCUSSION TITLE
Temporarily stop renovate bot during outage (Andon Cord)

### DISCUSSION THREAD

**Discussion by bahrmichael** [renovatebot/renovate#17378] [Request Help] [answered]
During an outage, we'd like to stop renovate from creating or merging pull requests (essentially doing anything). Preferrably without having to commit anything (e.g. removing the renovate.json file), but instead with a config checkbox (e.g. on the dependency dashboard) that we can check.

Is there a way to do this already, or a trick we can use?

If it's not possible to fully stop renovate, then preventing merges would be enough. We utilize `automerge` and `platformAutomerge`, but I guess we can only change that with a commit.

We're using the bot, not self-hosting.

**Comment by rarkins** [comment, marked as answer]
How are you running Renovate?

If you're using the hosted app then see https://docs.github.com/en/developers/apps/managing-github-apps/suspending-a-github-app-installation

**Reply by aisamu** [reply]
> How are you running Renovate?

We're [^1] using the hosted Github.com app
[^1]: Disclosure: colleague

**Reply by rarkins** [reply]
Suspending the app is definitely the best way. Once you unsuspend it, it should resume like before

**Reply by bahrmichael** [reply]
We're using the bot, not self-hosting

**Reply by rarkins** [reply]
So see the link on suspension above

**Reply by bahrmichael** [reply]
If I understand it correctly only the owner of an organization can suspend apps, right? We can page owners internally, but it would be nice if folks who are not owners of a GitHub org could stop renovate as well.
